tests/chainer_tests/test_link.py

```diff
@@ -345,6 +358,14 @@ def test_addgrads(self):
         numpy.testing.assert_array_equal(self.l2.x.grad, numpy.zeros(2))
         numpy.testing.assert_array_equal(self.l3.x.grad, numpy.zeros(3))
 
+    def test_serialize(self):
+        serializer = mock.MagicMock()
+        self.c1.serialize(serializer)
+
+        self.assertEqual(serializer.call_count, 0)
+        serializer['l1'].assert_called_once('x', self.l1.x.data)
+        serializer['l2'].assert_called_once('x', self.l2.x.data)
```

Comment:
Magicmock does not have `assert_called_once` method. Use `assert_called_once_with`. Travis CI does not raise errors  in Python 2.7 and 3.4 since `assert_called_once` is regarded as an attribute.
Recent mock module raises an attribute error when the attribute is missing and it starts with `assert` or `assret` . This seems the reason why Python 3.5 raised an error.

https://github.com/testing-cabal/mock/commit/7c530f0d9aa48d2538501761098df7a5a8979a7d